.github/workflows/pull_request.yml

```diff
@@ -3,13 +3,7 @@
 name: Pull Request
 
 on:
-  # Trigger this workflow on all pull request types, and push action is only for feature/chore/bug branches
   pull_request:
-    types: [ assigned, opened, synchronize, reopened, ready_for_review ]
-  push:
-    branches-ignore:
-      - develop
-      - 'release/**'
```

Comment:
lol it's a design so you can run the CD flow later in another action. Also, the push will help you recheck your branch before you open the PR 😛 anyway, it's a personal favor

Reply:
I think it's kindly spam, I prefer to make a githook to check before we push something 😁